src/encoderWorker.js

```diff
@@ -70,7 +70,9 @@ var OggOpusEncoder = function( config, Module ){
     numberOfChannels: 1,
     originalSampleRate: 44100,
     resampleQuality: 3, // Value between 0 and 10 inclusive. 10 being highest quality.
-    serial: Math.floor(Math.random() * 4294967296)
+    serial: Math.floor(Math.random() * 4294967296),
+    streamOpusPackets: true,
```

Comment:
since this is in the `OggOpusEncoder` class, we could just call this `streamPackets`

Reply:
I think that would be confused with "streaming ogg opus packets", which we aren't doing